index.yaml

```diff
@@ -3608,6 +3608,38 @@ entries:
     urls:
     - https://github.com/scalar-labs/helm-charts/releases/download/scalardl-audit-2.6.0/scalardl-audit-2.6.0.tgz
     version: 2.6.0
+  - apiVersion: v2
+    appVersion: 3.7.2
+    created: "2023-04-20T18:15:25.817201287+09:00"
+    dependencies:
+    - condition: envoy.enabled
+      name: envoy
+      repository: https://scalar-labs.github.io/helm-charts
+      version: ~2.2.0
+    description: ScalarDL is a tamper-evident and scalable distributed database. This
+      chart adds an auditing capability to Ledger (scalardl chart).
+    digest: 5528842a300a506a1d8b3664c83b961c7d35dc94c76306d91ef58d496ebcbf4c
+    home: https://scalar-labs.com/
+    icon: https://scalar-labs.com/wp-content/themes/scalar/assets/img/logo_scalar.svg
+    keywords:
+    - database
+    - distributed ledger
+    - scalar
+    - scalardl
+    maintainers:
+    - email: yusuke.morimoto@scalar-labs.com
```

Comment:
Can you update it with your name and email?
Yusuke and Kun can be deleted.

Reply:
Ah, OK. I will fix it. 
But, this `index.yaml` file is automatically generated based on the `Chart.yaml` file.
So, we should update `Chart.yaml` file instead of updating this `index.yaml` file directly.

I will do it in another PR!

Reply:
So, should we merge it for now?

Reply:
Yes. I will fix maintainers information as a bug fix in each chart. It will be applied in the next patch releases.
So, I will merge this PR.

Reply:
I did it in the following PRs.

https://github.com/scalar-labs/helm-charts/pull/200
https://github.com/scalar-labs/helm-charts/pull/201
https://github.com/scalar-labs/helm-charts/pull/202
https://github.com/scalar-labs/helm-charts/pull/203
https://github.com/scalar-labs/helm-charts/pull/204
https://github.com/scalar-labs/helm-charts/pull/205
https://github.com/scalar-labs/helm-charts/pull/206